tests/h/views/admin/groups_test.py

```diff
@@ -377,16 +379,6 @@ def pyramid_request(pyramid_request, factories, authority):
     return pyramid_request
 
 
-@pytest.fixture
-def paginate(patch):
-    return patch("h.views.admin.groups.paginator.paginate")
```

Comment:
Patches have either gone to the local test they relate to or the end of the file